src/test/e2e/helpers/test-helper.ts

```diff
@@ -45,15 +59,33 @@ function httpApis(
 
     return {
         createFeature: (
-            name: string,
+            feature: string | CreateFeatureSchema,
             project: string = DEFAULT_PROJECT,
             expectedResponseCode: number = 201,
         ) => {
+            let body = feature;
+            if (typeof feature === 'string') {
+                body = {
+                    name: feature,
+                };
+            }
             return request
                 .post(`${base}/api/admin/projects/${project}/features`)
-                .send({
-                    name,
-                })
+                .send(body)
+                .set('Content-Type', 'application/json')
+                .expect(expectedResponseCode);
+        },
+
+        getFeatures(
+            name?: string,
+            project?: string,
+            expectedResponseCode: number = 200,
+        ): supertest.Test {
+            const projectSubPath = project ? `projects/${project}/` : '';
+            const featureSubPath = name ? `/${name}` : '';
+            const featuresUrl = `/api/admin/${projectSubPath}features${featureSubPath}`;
```

Comment:
This follows REST conventions of having the feature resource and being able to get a single one of them. Alternatively, we could have two methods, but again, because this is test code, I thought it was better to simplify.

Features and project features responses are different (the first one does not return environments) and I only discovered this difference because of using these functions

Reply:
I decided to split the two APIs to make it clear that they're not the same: [8c9edf4](https://github.com/Unleash/unleash/pull/3348/commits/8c9edf444498e14f2931262ee654a040a6e526eb)